hudi-common/src/main/java/org/apache/hudi/avro/HoodieAvroUtils.java

```diff
@@ -437,6 +437,74 @@ public static Object getNestedFieldVal(GenericRecord record, String fieldName, b
     }
   }
 
+  /**
+   * Obtain the list of value , which read from multi fields.
```

Comment:
I'm not fully following what this method is doing, can you please explain with an example ?